packages/@aws-cdk/aws-lambda-nodejs/test/bundling.test.ts

```diff
@@ -647,7 +620,7 @@ test('esbuild bundling with pre compilations with undefined tsconfig ( Should fi
       command: [
         'bash', '-c',
         [
-          'tsc --project /asset-input/tsconfig.json --rootDir ./ --outDir ./ &&',
+          'tsc /asset-input/test/bundling.test.ts --rootDir ./ --outDir ./  &&',
```

Comment:
Where are the options from the tsconfig here? Why don't we see them?

Also it would nice to test with local bundling (without `forceDockerBundling`).